eip_operator/src/eip.rs

```diff
@@ -173,6 +177,51 @@ pub mod v2 {
                 _ => false,
             }
         }
+        pub fn get_resource_api(&self, client: &Client) -> Api<DynamicObject> {
+            match self.spec.selector {
+                EipSelector::Pod { pod_name: _ } => {
+                    let gvk = GroupVersionKind::gvk(
+                        Pod::group(&()).as_ref(),
+                        Pod::version(&()).as_ref(),
+                        Pod::kind(&()).as_ref(),
+                    );
+                    let api_resource = ApiResource::from_gvk(&gvk);
+                    // Api::namespaced(client, &self.namespace().unwrap())
+                    Api::default_namespaced_with(client.clone(), &api_resource)
```

Comment:
This should not be in the default namespace. This should be in the namespace of the EIP.